Enigmatry.Entry.Blueprint.Domain/Identity/UserQueryableExtensions.cs

```diff
@@ -1,9 +1,15 @@
 ﻿using Enigmatry.Entry.Blueprint.Domain.Users;
+using Microsoft.EntityFrameworkCore;
 
 namespace Enigmatry.Entry.Blueprint.Domain.Identity;
 
 public static class UserQueryableExtensions
 {
+    public static IQueryable<User> BuildAggregateInclude(this IQueryable<User> query) =>
```

Comment:
On Windesheim we use this naming convention in cases where we need to load entity with its related children to make root aggregate complete (this is usually used for the updates, not queries)